api/src/main/resources/workbench.yaml

```diff
@@ -36,6 +36,12 @@ paths:
             type: "array"
             items:
               $ref: "#/definitions/Cohort"
+      security:
+        - aou_oauth: []
+securityDefinitions:
+  aou_oauth:
+    type: oauth2
```

Comment:
What exactly do these do / how do they tie in with google sign-in? Does it affect server behavior at all? Can you add a comment?